### PR TITLE
Manual: move all git info to CONTRIB, explain how to use different branch in PR.

### PR DESCRIPTION
**Pull request by ericonr**
A little bit of reorganizing, and took @Chocimier's command from https://github.com/void-linux/void-packages/pull/23695#issuecomment-663226818

### CODE REVIEW

**Comment by Chocimier** [comment]
Please save description of actions in order of their execution.

"Fork a repo" does not describe how to send PR.

"git push --force" paragraph should be put in "Review" chapter. New branch is way to go when usual rewriting history caused troubles. Could you add something like following?

> More powerful way to modify commits than `git commit --amend` is [`git rebase -i HEAD~9`](https://git-scm.com/docs/git-rebase#_interactive_mode). It allows to join, reorder, change description of past commits and more.
    Alternatively, you can make other branch and push it to existing PR:
        git checkout master
        git checkout -b \<attempt2>
        # do changes anew
        git push -f \<fork> \<attempt2>:\<branch of pr>

Again: having minimal description of how to share done work at end of manual looks reasonable to me.

**Comment by ericonr** [comment]
I tried to include your suggestions, thanks!

Could I just repeat the forking info in the Manual? I mostly wanted to avoid duplication.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -101,6 +114,14 @@ We recommend having only a single commit for pull request, so if you need to mak
     $ git add <file>
     $ git commit --amend
     $ git push -f
+    
+A more powerful way of modifying commits than `git commit --amend` is with `git rebase -i HEAD~9`. It allows you to join, reorder, change description of past commits and more.
+
+Alternatively, if there are issues with your git history, you can make another branch and push it to the existing PR:
+
+    $ git checkout master -b <attempt2>
```
Comment (Chocimier) [inline CONTRIBUTING.md:122]: Oh nice, it's obvious what it does when you see it, but I didn't know it's possible.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -101,6 +114,14 @@ We recommend having only a single commit for pull request, so if you need to mak
     $ git add <file>
     $ git commit --amend
     $ git push -f
+    
+A more powerful way of modifying commits than `git commit --amend` is with `git rebase -i HEAD~9`. It allows you to join, reorder, change description of past commits and more.
```
Comment (Chocimier) [inline CONTRIBUTING.md:118]: Please link https://git-scm.com/docs/git-rebase#_interactive_mode here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,6 +16,19 @@ For this guide, we assume you have basic knowledge about [git](http://git-scm.or
 
 Please note that we do not accept any packages containing non-release versions, such as specific git- or svn-revisions anymore.
 
+To get started, [fork](https://help.github.com/articles/fork-a-repo) the void-linux `void-packages` git repository on GitHub and clone it:
+
+    $ git clone git@github.com:<user>/void-packages.git
+
+Once you've made changes to your forked repository, you can [submit a pull
```
Comment (Chocimier) [inline CONTRIBUTING.md:23]: Nothing to submit at the moment.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -16,6 +16,19 @@ For this guide, we assume you have basic knowledge about [git](http://git-scm.or
...
+request](#starting-a-pull-request).
+
+To keep your forked repository up to date, setup the `upstream` remote
```
Comment (Chocimier) [inline CONTRIBUTING.md:26]: Not sure where it should go. End of "Starting a pull request"?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -101,6 +114,14 @@ We recommend having only a single commit for pull request, so if you need to mak
     $ git add <file>
     $ git commit --amend
```
Comment (Chocimier) [inline CONTRIBUTING.md:115]: Link to ~here from "Instead, just forcibly git push" would be useful.

**Comment by ericonr** [comment]
I think I dealt with all comments.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -48,6 +57,8 @@ Once you have built your template file or files, the commit message should have
 
 If you want to describe your changes in more detail, add an empty line followed by those details ([example](https://github.com/void-linux/void-packages/commit/f1c45a502086ba1952f23ace9084a870ce437bc6)).
 
+Once you've made changes to your forked repository, you can [start a pull request](#starting-a-pull-request).
```
Comment (Chocimier) [inline CONTRIBUTING.md:60]: This line is out of order and unnecessary, changes should be tested before submitting, as stated below. I see that this relates to following paragraph, but maybe reword that instead.

**Comment by Chocimier** [comment]
> It should be concise summary of process, at least fork + mention of contributing.md + PR.

Please add mention on PR in Manual, either as was or some other form.
